src/main/java/GeneralService/OnlineServicesCommunicationService.java

```diff
@@ -0,0 +1,51 @@
+package GeneralService;
+
+import onlineservices.services.WindowControl.WindowControlService;
+import org.eclipse.paho.client.mqttv3.*;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OnlineServicesCommunicationService {
+
+    private static final String MQTT_BROKER_URL = "tcp://broker.emqx.io:1883";
+    private static final String MQTT_CLIENT_ID = "OnlineServicesCommunicationService";
+    private static final String MQTT_WINDOW_SERVICE_TOPIC = "WindowControlService";
+    private MqttClient windowControlServiceClient;
+    private WindowControlService windowControlService;
+
+    public OnlineServicesCommunicationService() {
+    }
+
+    public void sendRequest(Request request) {
+        if (RequestType.WINDOW_CONTROL_SERVICE_REQUEST.equals(request.getRequestType())) {
+            sendWindowControlServiceRequest(request);
+        }
+    }
+
+    private void sendWindowControlServiceRequest(Request request) {
```

Comment:
find a way to make this more generic, if we have 20 services i don't want 20 methods in here, maybe extract behavior to a different class, move that into a type of classes package (new) and there could be created more classes to define the behavior of other services. ex. WindowControlServiceRequest, maybe create an interface to generalize behavior for all the service request types, make it more generic